.github/workflows/wheel_linux_x86_64.yml

```diff
@@ -162,6 +162,10 @@ jobs:
           CIBW_BEFORE_BUILD: |
             cat /etc/yum.conf | sed "s/\[main\]/\[main\]\ntimeout=5/g" > /etc/yum.conf
             pip install ninja cmake
+            yum clean all -y
+            yum install centos-release-scl-rh -y
+            yum install devtoolset-11-gcc-c++ -y
+            source /opt/rh/devtoolset-11/enable -y
 
```

Comment:
It looks as though the environment still uses the older compiler here. We can additionally force the new toolchain by adding another step after the `CIBW_BEFORE_BUILD` and before the `CIBW_TEST_REQUIRES` stages:

```
CIBW_ENVIRONMENT: PATH=/opt/rh/devtoolset-11/root/usr/bin:$PATH
```

Reply:
Added this before `CIBM_TEST_REQUIRES`